boa/vyper/contract.py

```diff
@@ -363,7 +363,7 @@ def _dealias(self, maybe_address):
             return maybe_address
 
     def get(self, truncate_limit=None):
-        if isinstance(self.typ, MappingType):
+        if is_type_t(self.typ, HashMapT):
```

Comment:
these are not right. `is_type_t` checks if the argument is a TYPE_T with the second argument as its typedef. this should probably be `if isinstance(self.typ, HashMapT)`